src/consensus/consensus.mli

```diff
@@ -0,0 +1,77 @@
+open Crypto
+open Protocol
+
+module Operation_map : Map.S with type key = Protocol.Operation.t
+
+type identity = {
+  secret : Secret.t;
+  key : Key.t;
+  t : Key_hash.t;
+  uri : Uri.t;
+}
+
+type state = private {
```

Comment:
why not type t?

Reply:
https://github.com/marigold-dev/deku/pull/674#discussion_r893895978